aspnetcore/blazor/hybrid/routing.md

```diff
@@ -33,45 +50,42 @@ using Microsoft.AspNetCore.Components.WebView;
 Add the event handler to the constructor of the `Page` where the `BlazorWebView` is constructed:
 
 ```csharp
-blazorWebView.ExternalNavigationStarting += 
-    (sender, externalLinkNavigationEventArgs) =>
+blazorWebView.UrlLoading += 
+    (sender, urlLoadingEventArgs) =>
     {
-        externalLinkNavigationEventArgs.ExternalLinkNavigationPolicy = 
-            ExternalLinkNavigationPolicy.InsecureOpenInWebView;
+        urlLoadingEventArgs.UrlLoadingStrategy = UrlLoadingStrategy.OpenInWebView;
```

Comment:
Nit: Since we have a warning about opening external links in the WebView, maybe we should make the examples here do something else? For example, cancel all external navigation requests:

```suggestion
    if (urlLoadingEventArgs.UrlLoadingStrategy == UrlLoadingStrategy.OpenExternally)
            urlLoadingEventArgs.UrlLoadingStrategy = UrlLoadingStrategy.CancelLoad;
```

However, I realize this suggestion doesn't completely relate to the changes in this PR, and it does slightly complicate the example, so it might not be necessary. Just an idea.

Reply:
I leave the choice up to you; but we'll need to explain this in the text if the doc shows it.

Reply:
Let's leave it as-is then to keep the focus on how to subscribe to the event, rather than the logic inside the event handler.